src/App.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './App.css';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { fetchCrypto } from './redux/crypto/crypto';
+import Details from './components/Details';
+import Home from './components/Home';
+import Navigation from './components/Navbar';
+
+function App() {
+  const dispatch = useDispatch();
+  dispatch(fetchCrypto());
+  return (
+    <div className="App">
+      <Router>
+        <Navigation />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/details" element={<Details />} />
+        </Routes>
+      </Router>
+    </div>
+  );
```

Comment:
#### Dear @lawrahkonwea, Though you have a great design already as a front-end engineer we need to adapt to stick to the given design and deliver something identical. Please check out the following insight I have. 
- [ ] Notice how cards are arranged in two rows on the original design. We need to do the same here. 
- [ ] Please follow the original design on `typography`, `font face`, `size`, and `weight`.
- [ ] Layout composition and space between should look identical to the [Original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).

### [Original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).
![Screenshot 2023-02-24 203521](https://user-images.githubusercontent.com/69077061/221250436-52569027-54d7-4d48-9258-3a2ade9c9fcd.png)
### Your design.
![Screenshot 2023-02-24 203723](https://user-images.githubusercontent.com/69077061/221250448-cf939aa1-d7d6-4643-83c6-c54aef54da53.png)